eq_translations/extractable_strings.py

```diff
@@ -65,6 +65,16 @@
         "context": "Content",
         "additional_context": ["ListHeading", "ListDescription"],
     },
+    {
+        "json_path": "$..content.contents[*].definition.title",
+        "description": "Definition title",
+        "context": "Content",
+    },
+    {
+        "json_path": "$..content.contents[*].definition.contents[*].description",
+        "description": "Definition description",
+        "context": "Content",
+    },
```

Comment:
Examples of these should be added to the test schema with the correct extraction and translation files.

Reply:
We are really erratic at doing that

Reply:
We missed like 1 or 2, but we do them if we don't forgot. Even if there is no automated test, it highlights the extraction of the pot and transactions but the package is released. There is 1 test that extracts all translate items from the test schema and asserts that they are of the correct type.

Reply:
1 or 2 is being generous, there isn't a single content block in test_languages.json and very few block types

Reply:
We used to have a schema to have all extractable strings. We must have removed it. Maybe we just need a tidy up card for this remove so 👍